zipline/utils/calendars/calendar_helpers.py

```diff
@@ -73,7 +73,7 @@ def next_scheduled_day(date, last_trading_day, is_scheduled_day_hook):
         dt += delta
         if is_scheduled_day_hook(dt):
             return dt
-    return None
+    raise NoFurtherDataError(msg='Cannot find next day after %s' % date)
```

Comment:
I found one that needed to have the error handled, because there is a case where it might expect `None`. There's a bunch of other ones that would crash anyway if `None` was returned. Those don't need to be handled, correct?